front/app/containers/IdeasShowPage/index.tsx

```diff
@@ -1,41 +1,23 @@
 import React from 'react';
-import { isError } from 'lodash-es';
-import { isNilOrError } from 'utils/helperUtils';
 import { useParams } from 'react-router-dom';
 
 // components
+import { Box, useWindowSize } from '@citizenlab/cl2-component-library';
```

Comment:
Would be nice to replace useWindowSize with useBreakpoint? (perhaps out of scope here as I see you didn't actually touch that part).